.travis.yml

```diff
@@ -1,10 +1,11 @@
+dist: xenial
```

Comment:
out of curiosity, what's the need for changing the distro?

Reply:
See comment about xenial [here](https://github.com/LiliC/travis-minikube/blob/master/.travis.yml) (which is what I based these changes on).